DfE.FindInformationAcademiesTrusts/appsettings.json

```diff
@@ -14,8 +14,16 @@
     "ClientSecret": "",
     "TenantID": "",
     "RedirectUri": "",
+    "AllowedRoles": "fiat.create",
```

Comment:
Do we definitely need this line? We don't have different user roles in our application

Reply:
we dont need it ill remove